### PR TITLE
chore: Update node.js versions for travis

### PR DESCRIPTION
**Pull request by benurb**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -3,14 +3,8 @@ os:
...
-  - "lts/*"
-  - "4"
-
-before_script:
-  # Fixes an annoying bug in ESLint on Node 4
-  # https://github.com/babel/babel-eslint/issues/558
-  # TODO: Remove when Node 4 is not supported anymore
-  - npm install estraverse
+  - "8"
+  - "6"
```
Comment (jhnns) [inline .travis.yml]: Why not `lts/*`? 😁

Comment (benurb) [reply .travis.yml]: I had that first 😆 
Might be my personal preference, but I refer to it as "Node.js 8" and not "Node.js Carbon". We can change that ofc, but I guess more people have to look up codenames whereas version is pretty straight forward to understand 😄

**Comment by benurb** [comment]
See review comment.

File: .travis.yml (hunk 1 above)
Comment (benurb) [reply .travis.yml]: Ah, sorry. I guess I overinterpreted your question.
`lts/*` would only cover Node.js 8 currently and `""node", lts/*", "6"` just looked wrong 😆

Comment (jhnns) [reply .travis.yml]: Makes sense 👍

**Pull request merged**